src/pm/tui/screens/portfolio.py

```
@@ -0,0 +1,892 @@
+            return
+
+        if event.state == WorkerState.SUCCESS and self._loading_suggestions:
+            self._loading_suggestions = False
+            result = event.worker.result
+            if result:
+                self._ai_suggestions = result
+                try:
+                    detail = self.query_one(DetailPanel)
+                    detail.set_ai_suggestions(self._ai_suggestions)
+                except Exception:
+                    pass
```

Ruff (0.15.7)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)